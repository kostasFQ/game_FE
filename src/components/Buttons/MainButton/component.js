import React from 'react';
import styles from 'components/Buttons/Buttons.module.scss';
import PropTypes from 'prop-types';

const MainButton = ({ onClick }) => <button onClick={onClick} className={styles.mainButton}>CLICK HERE!</button>;

MainButton.propTypes = {
  onClick: PropTypes.func.isRequired,
}

export default MainButton;