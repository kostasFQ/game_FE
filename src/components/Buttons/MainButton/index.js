import MainButton from './MainButtonComponent'
export default MainButton;