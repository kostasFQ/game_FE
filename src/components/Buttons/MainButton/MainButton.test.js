import React from 'react';
import MainButton from './component';
import renderer from 'react-test-renderer';

test('MainButton should be render', () => {
  const button = renderer.create(
    <MainButton title='Click Here!' onClick={ () => {} } />
  );
  let component = button.toJSON();
  expect(component).toMatchSnapshot();
})