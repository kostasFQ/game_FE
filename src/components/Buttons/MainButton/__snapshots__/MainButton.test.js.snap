// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainButton should be render 1`] = `
<button
  className="mainButton"
  onClick={[Function]}
>
  CLICK HERE!
</button>
`;
